robot-core/src/main/java/org/obolibrary/robot/ReasonerHelper.java

```diff
@@ -46,9 +46,8 @@ public class ReasonerHelper {
   public static Set<OWLObjectProperty> getUnsatisfiableObjectProperties(OWLReasoner reasoner) {
     Set<OWLObjectProperty> unsatObjectProps = new HashSet<>();
 
-    if (reasoner
-        .getPrecomputableInferenceTypes()
-        .contains(InferenceType.OBJECT_PROPERTY_HIERARCHY)) {
+    if (reasoner.getPrecomputableInferenceTypes().contains(InferenceType.OBJECT_PROPERTY_HIERARCHY)
+        && !reasoner.getClass().getName().equals("org.semanticweb.elk.owlapi.ElkReasoner")) {
```

Comment:
This is necessary because of https://github.com/liveontologies/elk-reasoner/issues/66#issuecomment-1667697830

Reply:
Thanks @matentzn for tracking down the cause and for this solution!  I'm glad that it didn't require reverting the faster inference when using Hermit - that's been a huge boon in my workflow.